app/sections/judgeme-reviews/review-form.tsx

```diff
@@ -39,151 +44,190 @@ export function ReviewForm() {
   };
 
   return (
-    <div className="lg:w-1/3 md:w-2/5 w-full flex flex-col gap-5">
-      <div className="flex flex-col items-start gap-4 bg-line/30 p-6">
-        <p className="uppercase font-bold text-lg mb-1.5">
-          product reviews ({judgemeReviews.reviewNumber})
-        </p>
-        <div className="flex justify-start items-center gap-3">
-          <h4 className="font-medium">{judgemeReviews.rating.toFixed(1)}</h4>
-          <div className="flex gap-0.5">
-            <StarRating rating={judgemeReviews.rating} />
+    <div
+      className={`${
```

Comment:
Please use `clsx` for class construction

Reply:
yes i did it